package.json

```diff
@@ -34,5 +36,7 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
-}
+  },
+  "keywords": [],
+  "description": ""
```

Comment:
@harsha14gupta  Put a description here - `Official website of Girl Code It`. Don't leave it blank.

CC: @Manvityagi